adoc/chapters/programming_interface.adoc

```diff
@@ -2748,12 +2748,19 @@ struct execution_capabilities {
 } // namespace sycl::info::device
 ----
 
+Deprecated by SYCL {SYCL_VERSION}.
```

Comment:
Deprecated in SYCL 2020

Reply:
Fixed in 0e4149691d11786bcfc58464e78600622713d78e and e49d70cd618420a5842d4de16ba8209289427584